Yoast/wordpress-seo
# Move and improve metabox-editor-test

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* N/A

## Test instructions

This PR can be tested by following these steps:

* Run `composer test` and see the tests pass.

## UI changes

## Quality assurance


Fixes #
